glambie/util/unit_conversion_helpers.py

```diff
@@ -1,186 +1,227 @@
-def meters_to_gigatonnes(meters_list: list, rgi_area_km2: float, density_of_ice_in_gt_per_m3: float = 850) -> list:
+from typing import Iterable
+
+
+def meters_to_gigatonnes(variables: Iterable, area: float, density_of_ice: float = 850) -> list:
     """Function to convert a list of measurements of surface elevation change in meters into ice mass in gigatonnes,
     using the area of a region and the density of ice.
 
     Parameters
     ----------
-    meters_list : list
+    variables : Iterable
         A list of measurements in meters
-    rgi_area_km2 : float
+    area : float
         The area of a region in km2
-    density_of_ice_in_gt_per_m3 : float, optional
+    density_of_ice : float, optional
         The density of ice in Gt per m3, by default 850
 
     Returns
     -------
     A list of measurements in gigatonnes
     """
-    return [meter2gigatonne(i, rgi_area_km2, density_of_ice_in_gt_per_m3) for i in meters_list]
+    return [_convert_meters_to_gigatonnes(i, area, density_of_ice) for i in variables]
```

Comment:
in retrospect, all internal functions can be called _meters_to_gigatonnes etc. The conversions are implicit from module name